modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -59,16 +61,44 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 
 						<c:choose>
 							<c:when test='<%= FeatureFlagManagerUtil.isEnabled("LPS-188060") %>'>
-								<clay:dropdown-menu
-									cssClass="c-mr-3"
-									displayType="primary"
-									dropdownItems="<%= editKBArticleDisplayContext.getEditKBArticleActionDropdownItems() %>"
-									icon="caret-bottom"
-									id='<%= liferayPortletResponse.getNamespace() + "publishDropdown" %>'
-									label="<%= editKBArticleDisplayContext.getPublishButtonLabel() %>"
-									name="publishDropdown"
-									small="<%= true %>"
-								/>
+								<c:choose>
+									<c:when test="<%= kbArticle.isScheduled() %>">
+
+										<%
+										String displayDateString = StringPool.BLANK;
+
+										if (kbArticle.getDisplayDate() != null) {
+											displayDateString = dateFormatDateTime.format(kbArticle.getDisplayDate());
+										}
```

Comment:
Not 100% sure, but I think the display date cannot be `null`? Isn't it initialized to the creating date if omitted?

Reply:
I think you are right, I will check and remove that condicional if it is not needed.